Dockerfile

```diff
@@ -1,26 +1,25 @@
 FROM node:lts-alpine3.19 as builder
 
-COPY . ./
+COPY src ./src
+COPY docs ./docs
+COPY package.json .
+COPY package-lock.json .
+COPY tsconfig.json .
 
 RUN npm install --legacy-peer-deps --omit=dev &&\
     npm cache clean --force &&\
-    npm run compile &&\
-    npm test &&\
-    npm run build:docs
+    npm run build:docs &&\

```

Comment:
does this also build the eslint app? (that we then copy from /dist)

Reply:
yes. The build:docs script builds before generating the docs to make sure the latest changes are in place.

> "build": "tsc --showConfig && tsc",

> "build:docs": "npm run build && node dist/src/docGeneratorMain.js",